pkg/osquery/tables/wlan/wlan.go

```diff
@@ -0,0 +1,198 @@
+package wlan
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"os/exec"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/osquery-go"
+	"github.com/kolide/osquery-go/plugin/table"
+	"github.com/pkg/errors"
+)
+
+const netshCmd = "netsh"
+
+var preludeInterfaceLineRegex = regexp.MustCompile(`Interface name :`)
+var preludeCountLineRegex = regexp.MustCompile(`There are [0-9]+ networks currently available`)
+
+type execer func(ctx context.Context, buf *bytes.Buffer) error
```

Comment:
As a general note, the style of passing in a bytes.Buffer is considered higher performance (yay).

But, the underlying call is an os level exec, which is always going to pass a bunch of data around weirdly. I've generally biased to things that returned bytes. Lower performance, but easily swappable with something like `ioutil.ReadFile`